datasets/id_panl_bppt/README.md

```diff
@@ -0,0 +1,143 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+languages:
+- en
+- id
+licenses: []
+multilinguality: 
+- translation
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
+---
+
+# Dataset Card for [Dataset Name]
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [PANL BPPT](http://digilib.bppt.go.id/sampul/p92-budiono.pdf)
+- **Repository:** [PANL BPPT Repository](https://github.com/cahya-wirawan/indonesian-language-models/raw/master/data/BPPTIndToEngCorpusHalfM.zip)
+- **Paper:** [Resource Report: Building Parallel Text Corpora for Multi-Domain Translation System](http://digilib.bppt.go.id/sampul/p92-budiono.pdf)
+- **Leaderboard:**
+- **Point of Contact:**
+
+### Dataset Summary
+Parallel Text Corpora for Multi-Domain Translation System created by BPPT (Indonesian Agency for the Assessment and 
+Application of Technology) for PAN Localization Project (A Regional Initiative to Develop Local Language Computing 
+Capacity in Asia). The dataset contains around 24K sentences divided in 4 difference topics (Economic, international,
+Science and Technology and Sport).
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+Indonesian
+
+## Dataset Structure
+
+[More Information Needed]
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+- `id`: id of the sample
+- `translation`: the parallel sentence english-indonesian
+- `topic`: the topic of the sentence
```

Comment:
can you give more details about the values that `topic` can take ?

Reply:
a topic list has been added